### PR TITLE
IfcElement Rail Import BIMQ 2021-07-21T07:43:32.1157664Z

### PR DESCRIPTION
**Pull request by jmirtsch**
Rail properties for IfcElement

### CODE REVIEW

File: IFC4x3/Properties/p/PuttingIntoOperationDate_03cT7VFl92dw8n1GPuqXSv/Documentation.md [hunk 1]
```diff
@@ -0,0 +1 @@
+Date on which the banquet element was (or will be) put into operation.
```
Comment (AlexBrad1eyCT) [inline IFC4x3/Properties/p/PuttingIntoOperationDate_03cT7VFl92dw8n1GPuqXSv/Documentation.md:1]:
```suggestion
Date on which the element was (or will be) put into operation.
```

File: IFC4x3/Properties/d/DisassemblyDate_21F9eF2br0OPwbIT8YD7$C/Documentation.md [hunk 2]
```diff
@@ -0,0 +1 @@
+Date on which the banquet element was (or will be) removed.
```
Comment (AlexBrad1eyCT) [inline IFC4x3/Properties/d/DisassemblyDate_21F9eF2br0OPwbIT8YD7$C/Documentation.md:1]:
```suggestion
Date on which the element was (or will be) removed.
```

**Review by AlexBrad1eyCT — CHANGES_REQUESTED** [review]
No Issues with this extension to the ManufacturerOccurance Pset I agree the additions are necessary and currently absent from the schema.

Just tweak the documentation suggestions and i will set it to Approve

**Comment by AlexBrad1eyCT** [comment]
See Comments in #171 which will impact changes in this PR

**Review by AlexBrad1eyCT — APPROVED** [review]
LFTM, but related comments in #171 need to be addressed before merging and a decision on which PR to submit them in

File: IFC4x3/Properties/i/IK_Code_2kpG9_rK93LwNz4pCQcRnH/DocProperty.xml [hunk 3]
```diff
@@ -1,3 +1,3 @@
 <?xml version="1.0" encoding="utf-8"?>
-<DocProperty xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" id="IK_Code_2kpG9_rK93LwNz4pCQcRnH" Name="IK_Code" UniqueId="aecd027e-d542-4357-a5fd-13331a99bc51" />
```
Comment (SergejMuhic) [inline IFC4x3/Properties/i/IK_Code_2kpG9_rK93LwNz4pCQcRnH/DocProperty.xml:2]: Marking for removal.

File: IFC4x3/Properties/p/PuttingIntoOperationDate_03cT7VFl92dw8n1GPuqXSv/DocProperty.xml [hunk 4]
```diff
@@ -0,0 +1,3 @@
+<?xml version="1.0" encoding="utf-8"?>
+<DocProperty xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" id="PuttingIntoOperationDate_03cT7VFl92dw8n1GPuqXSv" Name="PuttingIntoOperationDate" UniqueId="0399d1df-3ef2-429f-a231-050678d21739" PrimaryDataType="IfcDate" />
```
Comment (SergejMuhic) [inline IFC4x3/Properties/p/PuttingIntoOperationDate_03cT7VFl92dw8n1GPuqXSv/DocProperty.xml:2]: I would propose a renaming to `StartOperationDate`. And a question, does this really fit into Manufacturer Occurence? Looking forward to a discussion.

**Review by SergejMuhic — COMMENTED** [review]
I guess some of these are debatable if they should be in manufacturer occurrence. The description of the pset says "Defines properties of individual instances of manufactured products that may be given by the manufacturer." Are all of these dates given by the manufacturer?

File: IFC4x3/Properties/p/PuttingIntoOperationDate_03cT7VFl92dw8n1GPuqXSv/DocProperty.xml (hunk 4 above)
Comment (czapplitec) [reply IFC4x3/Properties/p/PuttingIntoOperationDate_03cT7VFl92dw8n1GPuqXSv/DocProperty.xml]: This one is removed from this pull request. All these "XxxDate" will be managed together in another pull request.

**Pull request merged**